README.md

```diff
@@ -238,3 +238,43 @@ Notes:
 * This API is not defined by O2ims Interface Specification.
 * The server supports the `alarmProbableCauses` endpoint for exposing a custom list of probable causes.
 * The list is available in [data folder](internal/files/alarms/probable_causes.json). Can be customized and maintained as required.
+
+#### Alarm Subscription server
+
+The alarm subscription server needs to connect to the non-kubernetes API of the
+ACM global hub. To use the configmap to persist the subscriptions, the namespace "orantest" should be created at hub cluster for now.
+
+Start the deployment manager server with a command like this:
+
+```
+$./oran-o2ims start alarm-subscription-server \
+--log-file="servers.log" \
+--log-level="debug" \
+--log-field="server=alarm-subscription" \
+--log-field="pid=%p" \
+--api-listener-address="127.0.0.1:8006" \
+--metrics-listener-address="127.0.0.1:8008" \
+--cloud-id="123" 
+```
+
+Note that by default all the servers listen on `localhost:8000`, so there will
+be conflicts if you try to run multiple servers in the same machine. The
+`--api-listener-address` and `metrics-listener-address` options are used to select a port number that isn't in use.
```

Comment:
nit: to keep convention, consider adding the '--' prefix to both, i.e. `--metrics-listener-address`.

Reply:
Thanks for the comments and Addressed. And add the option to other examples as well.